main.go

```diff
@@ -112,6 +112,26 @@ func main() {
 		log.Fatal("Failed to start the email finalizer, err: ", err)
 	}
 
+	// load NCMEC credentials
+	ncmecReportingEnabled := true
+	ncmecCredentials, err := email.LoadNCMECCredentials()
+	if err != nil {
+		log.Print("Failed to load NCMEC credentials", err)
+		ncmecReportingEnabled = false
```

Comment:
I would go for an approach like we do with TUS and Jaeger where you can enable or disable ncmec reporting with an env var but if it is enabled, we won't boot if we encounter an error due to missing env vars.

Reply:
yeah this didn't feel right, done 👍